e2e/interchaintestv8/ibc_eureka_test.go

```diff
@@ -114,20 +114,20 @@ func (s *IbcEurekaTestSuite) SetupSuite(ctx context.Context) {
 	}))
 
 	s.Require().True(s.Run("Deploy ethereum contracts", func() {
-		s.Require().NoError(operator.RunGenesis(
+		args := append([]string{
 			"--trust-level", testvalues.DefaultTrustLevel.String(),
 			"--trusting-period", strconv.Itoa(testvalues.DefaultTrustPeriod),
 			"-o", testvalues.Sp1GenesisFilePath,
-		))
+		}, pt.ToOpGenesisArgs()...)
+		s.Require().NoError(operator.RunGenesis(args...))
 
 		var (
 			stdout []byte
 			err    error
 		)
 		switch prover {
 		case testvalues.EnvValueSp1Prover_Mock:
-			stdout, err = eth.ForgeScript(s.deployer, "scripts/MockE2ETestDeploy.s.sol:MockE2ETestDeploy")
```

Comment:
Should we not have this support anymore? If so, we should probably just delete the MockE2ETestDeploy script as well and update the README accordingly?

Reply:
Yeah, because I removed the ability to override sp1 verifier. This can only be done in solidity tests using forge-std cheatcodes now